website/docs/prysm-usage/fee-recipient.md

```diff
@@ -0,0 +1,76 @@
+---
+id: fee-recipient
+title: Fee Recipient
+sidebar_label: fee recipient
+---
+
+# Fee Recipient
+
+Fee Recipient is the feature that allows users receive gas fees when proposing blocks. These gas rewards are some times know as the inclusion rewards.
+
+:::tip Post Bellatrix Fork ( The Merge )
+This feature will only activate Post Bellatrix Fork. Currently stakers do not receice gas fees from proposing blocks.
+:::
+
+:::warning Known Release Bug
+Setting a default fee recipient through the validator client is currently not working in the upcoming branch a fix was created and merged into develop for the following release. please refer to [#10555](https://github.com/prysmaticlabs/prysm/pull/10555)
+:::
+
+:::warning Be aware of your Fee Recipient Settings!!
+
+Protect and review your Fee Recipient settings.
+
+Fee Recipient settings will default to the burn address ( 0x000....0 ) so it is important to set it to a hex address where you can be paid this. 
+
+Please also make sure the address is yours when it is set, otherwise it will be sent to someone else's address.
+:::
+
```

Comment:
Recommend including some introductory text here along the lines of "This new feature introduces a configuration file that lives _________________. Your client will read this file when ______________. 

To configure Fee Recipient, first ____________________ (create the file? add configuration boilerplate to an existing file?). Then, adjust the settings as needed. The following settings can be configured: (insert a table here for each parameter / flag value with a brief description)."